samcli/commands/remote_invoke/remote_invoke_context.py

```diff
@@ -187,3 +189,29 @@ def _get_from_physical_resource_id(self) -> CloudFormationResourceSummary:
                     f"Please provide full resource ARN or --stack-name to resolve the ambiguity."
                 )
             return resource_summary
+
+    @property
+    def stdout(self) -> StreamWriter:
```

Comment:
Should this be more common than here?

Reply:
I added it to the context because if we decide to add another handle, for instance log_file option that `sam local` has, we can add that here directly after passing it to the context.